test/end-to-end/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "end-to-end",
```

Comment:
We probably want to name this something line `weldr-web-end-to-end-tests` to be more descriptive.